package.json

```diff
@@ -40,12 +40,13 @@
     "graphql-tag": "^2.12.6",
     "mustache": "^4.2.0",
     "object-path": "^0.11.8",
-    "subscriptions-transport-ws": "^0.11.0",
+    "subscriptions-transport-ws": "^0.9.19",
```

Comment:
Peer dependency for `apollo-link-ws@^1.0.20` is `subscriptions-transport-ws@^0.9.19`, not `0.11.0`. With newer version of npm, it will throw an error when there is a conflict, unless it is explicitly told to ignore the conflict by using `--legacy-peer-deps` flag.

I checked the release note for `0.11.0`, and downgrading to `0.9.19` should not cause issue.